web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -869,6 +869,10 @@ $.extend(fixmystreet.set_up, {
         }
         $('#key-tools li:empty').remove();
         $('#report-updates-data').insertAfter($('#map_box'));
+        if (fixmystreet.page === 'report') {
+            $('#sub_map_links').append('<a href="#" id="show-fullscreen" class="expand">Expand map</span>');
+            $('#sub_map_links').append('<a href="#" id="hide-fullscreen" class="compress hidden">Collapse map</span>');
```

Comment:
Would this be nicer as one link that changes class/text when clicked, rather than two? Guess it doesn't make much difference, but I think other similar uses in the code currently toggle the one link.